README.md

```diff
@@ -102,3 +102,50 @@ print(variant)
 ```
 
 For more information about variants, see the [Variant documentation](https://docs.getunleash.io/advanced/toggle_variants).
+
+## Developing
+
+For development, you'll need to setup the environment to run the tests. This repository is using
+tox to run the test suite to test against multiple versions of Python. Running the tests is as simple as running this command in the makefile:
+
+```
+make tox
```

Comment:
Might be better to use tox and a py version here otherwise I think this will try run this on all your supported python versions and spazz a little
```suggestion
tox -e py311
```